web/src/beta/components/Slider/index.tsx

```diff
@@ -1,52 +1,55 @@
 import RCSlider from "rc-slider";
 import React, { ComponentProps } from "react";
 
-import { styled, css } from "@reearth/services/theme";
+import { styled } from "@reearth/services/theme";
 
 import "rc-slider/assets/index.css";
 
-type Props = {
+export type Props = {
   min: number;
   max: number;
-  frame?: boolean;
 } & Omit<ComponentProps<typeof RCSlider>, "defaultValue">;
 
-const Slider: React.FC<Props> = ({ frame = false, ...props }) => (
-  <Wrapper frame={frame}>
+// TODO: Show tooltip on hover and drag
```

Comment:
To implement tooltip either I can build a small `custom component` or use `rc-tooltip` from the example at https://github.com/react-component/slider/blob/master/docs/examples/components/TooltipSlider.tsx ? 

Let me know which one works. I don't want to add a dependency for a small functionality but if it's going to be used over and over again it makes sense

Reply:
Go with the example you reference. rc-tooltip is a dep of rc-slider so you don't need to add any new dep 👍🏻

Reply:
Thanks! :+1: